content/streaming-api/index.textile

```diff
@@ -0,0 +1,115 @@
+
+This feature is currently in beta.
+
+It is subscribe-only: you can not interact with the channel, including to publish, enter presence, query the presence set, attach and detach from channels (without closing and re-opening the stream), or anything else.
+
+Where possible, we strongly recommend the use of one of our "Realtime client libraries":https://www.ably.io/download , which provide "more features and higher reliability":https://support.ably.io/solution/articles/3000061848-should-i-use-the-rest-api-directly , and the full use of our "normal realtime messaging API":/realtime.
+
+h2(#authentication). Authentication
+
+The same authentication options as used in the REST api also apply here. So for basic auth, a querystring parameter of @key@ or an @Authorization: Basic <base64-encoded key>@ header; for token auth, a querystring parameter of @accessToken@ containing an access token or an @Authorization: Bearer <base64-encoded token>@ header. See "REST API authentication":/rest-api#authentication for more information.
+
+
+h2(#connection-state-recovery). Connection state recovery
+
+If you are disconnected, you can supply the @id@ of the last event you received as the @lastEvent@ querystring parameter of a new connection, to resume from that point.
+
+Connection state is only retained for two minutes. See "Connection state explained":/realtime/connection#connection-state-explained for full documentation.
+
+Due to this feature being in beta, it is currently subject to other restrictions. For example, at the moment you can not resume from part-way through a backlog of messages that was being sent as part of a previous resume; you can only resume from a message (so you can't resume unless you've received at least one message on the connection); and you can only resume from the id of a message that was sent in the last two minutes (so it will not work if the last message sent on the connection was more than two minutes ago). We expect to fix all of these by the time this feature graduates out of beta.
+
+h1(#api). API routes
+
+h3(#event-stream). Plain event stream
+
+h6. GET realtime.ably.io/event-stream
+
+Start a streaming HTTP request.
+
+h5. Request parameters
+
+- channels := (mandatory) one or more channel names, separated by commas (or the @separator@ if specified). Non-url-safe characters should be URL-encoded (for example, @?channels=foo%3Fbar@ will subscribe to the channel @foo?bar@). Alias: @channel@
+- separator := (optional) a separator, to enable easy subscriptions to channels with commas in their name. For example, @?channels=?separator=|&channel=fo,o|ba,r@ will subcribe to the two channels @fo,o@ and @ba,r@.
+- v := (mandatory) the version of the api you are requesting. The current (and only) version of the API is 1.1, so only @v=1.1@ is accepted
+- key := (optional) an Ably API key to use, if using basic auth
+- accessToken := (optional) an Ably auth token to use, if using token auth
```

Comment:
Can this not be `token`?  Would JWT work, I assume so?

Reply:
Auth is processed by the same code (AuthMgr) whether for rest, websocket, or this. So JWT will work fine, and we can add `token` as an alias of `accessToken` (and `access_token`) if we want as long as nothing else uses that param (and I don't imagine anything does), sure

Reply:
> So JWT will work fine, and we can add token as an alias of accessToken

Ok, sounds good.  Given we have `key` as opposed to `accessKey`, I think using `token` is preferable

Reply:
Except that "access token" is widely known and accepted terminology, among the various other token types used in oauth schemes (client token, user token, refresh token etc). I don't see the advantage in adding an ambiguous `token` param.

Reply:
>  I don't see the advantage in adding an ambiguous token param.

Brevity and consistency with our client SDKs which only refer to tokens as `token` or `tokenDetails`

Reply:
So if we have both `accessToken` and `token`, which is the preferred or primary name, and which is the alias? In other situations we've argued against having multiple ways of doing the same thing, because it only creates confusion.

Reply:
> In other situations we've argued against having multiple ways of doing the same thing, because it only creates confusion.

My preference is just `token`